services/worker/src/worker/job_runners/config/parquet_and_info.py

```diff
@@ -741,33 +631,35 @@ def compute_config_parquet_and_info_response(
     Returns:
         `ConfigParquetAndInfoResponse`: An object with the config_parquet_and_info_response
           (dataset info and list of parquet files).
-    <Tip>
     Raises the following errors:
+        - [`libcommon.exceptions.AskAccessHubRequestError`]
+          if the request to the Hub to get access to the dataset failed or timed out.
+        - [`libcommon.exceptions.DatasetNotFoundError`]:
+          if the dataset does not exist, or if the token does not give the sufficient access to the dataset,
+        - [`libcommon.exceptions.GatedDisabledError`]
+          if the dataset is gated, but disabled.
+          or if the dataset is private (private datasets are not supported by the datasets server).
+        - [`libcommon.exceptions.GatedExtraFieldsError`]
+          if the dataset is gated, with extra fields. Programmatic access is not implemented for this type of
+          dataset because there is no easy way to get the list of extra fields.
+        - ['requests.exceptions.HTTPError'](https://requests.readthedocs.io/en/latest/api/#requests.HTTPError)
+          any other error when asking access
+        - [`libcommon.simple_cache.CachedArtifactError`]
+            If the previous step gave an error.
         - [`~job_runners.config.parquet_and_info.DatasetInBlockListError`]
```

Comment:
I think almost all the exceptions were moved to libcommon.exceptions ? There are still some comments pointing to job_runners.config.parquet_and_info.

Reply:
thanks. Fixed in https://github.com/huggingface/datasets-server/pull/1169/commits/60c9ea625af69c2c4790623dd021ffb189fe39f6